UDF/tg_ExprFunctions.hpp

```diff
@@ -0,0 +1,440 @@
+/******************************************************************************
+ * Copyright (c) 2015-2021, TigerGraph Inc.
+ * All rights reserved.
+ * Project: TigerGraph Query Language
+ * udf.hpp: a library of user defined functions used in queries.
+ *
+ * - This library should only define functions that will be used in
+ *   TigerGraph Query scripts. Other logics, such as structs and helper
+ *   functions that will not be directly called in the GQuery scripts,
+ *   must be put into "ExprUtil.hpp" under the same directory where
+ *   this file is located.
+ *
+ * - Supported type of return value and parameters
+ *     - int
+ *     - float
+ *     - double
+ *     - bool
+ *     - string (don't use std::string)
+ *     - accumulators
+ *
+ * - Function names are case sensitive, unique, and can't be conflict with
+ *   built-in math functions and reserve keywords.
+ *
+ * - Please don't remove necessary codes in this file
+ *
+ * - A backup of this file can be retrieved at
+ *     <tigergraph_root_path>/dev_<backup_time>/gdk/gsql/src/QueryUdf/ExprFunctions.hpp
+ *   after upgrading the system.
+ *
+ ******************************************************************************/
+
+#ifndef EXPRFUNCTIONS_HPP_
```

Comment:
This should probably have a different name from the regular ExprFunctions.hpp file. Maybe TG_EXPRFUNTIONS_HPP_